FreeAPS/Sources/Views/DecimalTextField.swift

```diff
@@ -85,7 +88,9 @@ public struct DecimalTextField: UIViewRepresentable {
             action: #selector(Coordinator.cancelEdit)
         )
 
-        toolbar.items = [clearButton, flexibleSpace, cancelButton, doneButton]
+        toolbar
+            .items = liveEditing ? [clearButton, flexibleSpace, doneButton] :
+            [clearButton, flexibleSpace, cancelButton, doneButton]
```

Comment:
don't show Cancel when liveEditing=true since the value is already updated and we can't cancel